docs/backend/native_api.ipynb

```diff
@@ -15,7 +15,8 @@
     "- `/flush_cache`\n",
     "- `/get_memory_pool_size`\n",
     "- `/update_weights`\n",
-    "- `/encode`\n",
+    "- `/encode`(embedding model)\n",
+    "- `/judge`(reward model)\n",
```

Comment:
rename `/generate` -> `/generate (text generation model)`

Reply:
![image](https://github.com/user-attachments/assets/d466fde2-7dfe-45bf-8543-a4a91a90ddbb)


i changed the order of our contents, to make it looks consistent. 😂

Reply:
it is better to order by frequency. `/generate` is the mostly widely used endpoint